spec/rubocop/cop/lint/eval_spec.rb

```diff
@@ -30,4 +30,9 @@
                    'something.eval')
     expect(cop.offenses).to be_empty
   end
+
```

Comment:
Yes, that's fine. I thought that adding these examples would require an update in the tested code, but I was mistaken. Still, it's good to cover these cases too.